logger.go

```diff
@@ -183,6 +188,16 @@ func findCaller() *CallerInfo {
 	}
 }
 
+type CtxFields map[string]string
+
+func (fields *CtxFields) save(k, v string) {
```

Comment:
Does the receiver need to be a pointer? Maps are already pointers.
And any reason not use `set`? (as opposed to `get`)